command/src/main/java/com/jagrosh/jdautilities/command/CommandBuilder.java

```diff
@@ -64,15 +64,13 @@
      * @param  name
      *         The name of the Command to be built.
      *
-     * @return This CommandBuilder
      */
-    public CommandBuilder setName(String name)
+    public void setName(String name)
```

Comment:
This is a breaking change and makes the builder less useful as you remove the fluent interface pattern for no reason.